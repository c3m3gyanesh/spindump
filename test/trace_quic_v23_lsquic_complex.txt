A more complex run with Lsquic client to Picoquic server. This succeeds in handshake, then some data is transmitted but one of the endpoints fail and the connection breaks
