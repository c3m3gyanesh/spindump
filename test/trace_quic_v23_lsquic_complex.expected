QUIC 172.16.95.160 <-> 34.238.92.221 08d60b0f1874b3c3-1f38d6bf29dc41721b9ea633e095e56eac5f88 (63324:4433) at 1571098275521281 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.95.160 <-> 34.238.92.221 08d60b0f1874b3c3-08de3011c87ea67c (63324:4433) at 1571098275595196 measurement up right 73915 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v23,no spin"
QUIC 172.16.95.160 <-> 34.238.92.221 d60b0f1874b3c3b1-08de3011c87ea67c (63324:4433) at 1571098275598582 spinvalue up 0 responder packets 2 3 bytes 1304 3778 bandwidth 1304 3778 note "V23,no spin"
QUIC 172.16.95.160 <-> 34.238.92.221 d60b0f1874b3c3b1-08de3011c87ea67c (63324:4433) at 1571098275599305 spinvalue up 0 initiator packets 2 4 bytes 1304 5176 bandwidth 1304 5176 note "V23,no spin"
QUIC 172.16.95.160 <-> 34.238.92.221 d60b0f1874b3c3b1-08de3011c87ea67c (63324:4433) at 1571098275815954 spinvalue up 0 responder packets 4 5 bytes 1657 5256 bandwidth 1657 5256 note "V23,no spin"
QUIC 172.16.95.160 <-> 34.238.92.221 d60b0f1874b3c3b1-08de3011c87ea67c (63324:4433) at 1571098276853407 spinvalue up 0 responder packets 4 6 bytes 1657 5661 bandwidth 1657 5661 note "V23,no spin"
QUIC 172.16.95.160 <-> 34.238.92.221 d60b0f1874b3c3b1-08de3011c87ea67c (63324:4433) at 1571098278837846 spinvalue up 0 responder packets 4 7 bytes 1657 6066 bandwidth 1657 5661 note "V23,no spin"
